anvil/src/udev.rs

```diff
@@ -513,17 +539,60 @@ fn scan_connectors(
                 .user_data()
                 .insert_if_missing(|| UdevOutputId { crtc, device_id });
 
-            let damage_tracked_renderer = DamageTrackedRenderer::from_output(&output);
             #[cfg(feature = "debug")]
             let fps_element = FpsElement::new(fps_texture.clone());
 
+            let driver = match device.get_driver() {
+                Ok(driver) => driver,
+                Err(err) => {
+                    warn!(logger, "Failed to query drm driver: {}", err);
+                    continue;
+                }
+            };
+
+            let mut planes = match surface.planes() {
+                Ok(planes) => planes,
+                Err(err) => {
+                    warn!(logger, "Failed to query surface planes: {}", err);
+                    continue;
+                }
+            };
+
+            // Using an overlay plane on a nvidia card breaks
+            if driver.name().to_string_lossy().to_lowercase().contains("nvidia")
```

Comment:
How many driver workarounds do we plan to have? If it's going to go beyond a few I think we should adopt some sort of WorkaroundFlags

Reply:
I don't know, but this workaround is in an anvil only, not smithay. But I can move it to a separate function If you want?